content/source/docs/enterprise/admin/agents-on-tfe.html.md

```diff
@@ -46,3 +46,7 @@ This includes the Terraform releases distribution service, [releases.hashicorp.c
 as well as the [Terraform provider registry](https://registry.terraform.io). Agents
 executing in a workspace that leverage a Terraform version that provides a custom
 Terraform bundle with pre-existing provider binaries do not need access to these resources.
+
+* **Agent Version Compatibility**: TFE places restrictions on what versions of Terraform Cloud Agents can be registered. This is to prevent an incompatible agent from registering with a TFE instance and attempting to execute a Terraform operation in an undefined way. Compatible versions of Terraform Cloud Agents on TFE will vary based on the specific TFE release sequence; any changes to compatible Terraform Cloud Agents versions will be noted in the [TFE release notes](https://github.com/hashicorp/terraform-enterprise-release-notes).
+
+-> **Note:** TFE is currently compatible with Terraform Cloud agents versions `0.4.2` and lower.
```

Comment:
i don't think leaving a 'current compatible' note here is a great strategy long-term, but because none of the _current_ TFE release notes mention compatibility, having this here for now seems like a reasonable tradeoff to provide a source of truth. in the next TFE release when compatibility with `1.x` agents is released, we will have an explicit mention in the TFE release notes, and we can remove this footnote at that time.